CONTRIBUTING.md

```diff
@@ -958,8 +958,9 @@ The pattern is simple:
    [cx-api/lib/features.ts](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cx-api/lib/features.ts).
    This map is inserted to generated `cdk.json` files for new projects created
    through `cdk init`.
-4. In your tests, use the `testFutureBehavior` and `testLegacyBehavior` [jest helper methods] to test the enabled and disabled behavior.
-5. In your PR title (which goes into CHANGELOG), add a `(under feature flag)` suffix. e.g:
+4. Add an entry for your feature flag in the [README](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cx-api/README.md) file.
```

Comment:
cc @TheRealAmazonKendra. I've started adding things here for the time being, but not sure what you think.

Reply:
No objections.